From c2e8bf5de19432d64d84565fa89392f13eb49286 Mon Sep 17 00:00:00 2001
From: TheGreatMcPain <james@thegreatmcpain.xyz>
Date: Mon, 2 Dec 2024 00:29:57 -0600
Subject: [PATCH 1/2] dts: add rk3399-linux.dtsi to rk3399-rockpro64.dtsi

Seems to fix a number of issues. Like the pwm-fan, the HDMI port, and
the hardware video decoder/encoder.
---
 arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dtsi
index 78157521e944..0024f926d9f4 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-rockpro64.dtsi
@@ -8,6 +8,7 @@
 #include <dt-bindings/pwm/pwm.h>
 #include "rk3399.dtsi"
 #include "rk3399-opp.dtsi"
+#include "rk3399-linux.dtsi"
 
 / {
 	aliases {
-- 
2.45.2

